test/plugin_functional/test_suites/core_plugins/rendering.ts

```diff
@@ -255,6 +255,7 @@ export default function ({ getService }: PluginFunctionalProviderContext) {
          * and will resolve to (boolean)
          */
         'xpack.infra.featureFlags.metricsExplorerEnabled (any)',
+        'xpack.infra.featureFlags.osqueryEnabled (any)',
```

Comment:
Could we add `osqueryEnabled` to the above comment (alongside `metricsExplorerEnabled`)explaining that it is conditional?